common/app/views/fragments/atoms/storyquestions.scala.html

```diff
@@ -3,31 +3,62 @@
 @if(!isAmp) {
     <div class="js-view-tracking-component submeta user__question">
         <span class="js-storyquestion-atom-id is-hidden" id="@storyquestions.id"></span>
+
+        @defining("test/test" == storyquestions.data.relatedStoryId && storyquestions.atom.labels.find(_.length == 4)) { isEmailSubmissionReady =>
```

Comment:
Wow, is Scala coercing an `Option[T]` into a `Boolean` here? I didn't know that was possible 😅